src/components/Tabs.js

```diff
@@ -1,235 +1,33 @@
-
-    if (this.props.isActive)
-      style = Object.assign({}, style, styles.activeTab, this.props.styles.activeTab);
-
-    return (
-      <span onClick={this.props.onClick} style={style}>
-        {this.props.children}
-      </span>
-    );
-  }
-
-  styles = () => {
-    return {
-      tab: {
-        boxSizing: 'border-box',
-        color: StyleConstants.Colors.ASH,
-        cursor: 'pointer',
-        display: 'inline-block',
-        fontSize: StyleConstants.FontSizes.MEDIUM,
-        fontStyle: StyleConstants.Fonts.SEMIBOLD,
-        padding: StyleConstants.Spacing.MEDIUM,
-        whiteSpace: 'nowrap',
-
-        ':hover': {
-          color: StyleConstants.Colors.CHARCOAL
-        }
-      },
-      activeTab: Object.assign({
-        cursor: 'default',
-        color: this.props.brandColor,
-        borderBottom: '2px solid ' + this.props.brandColor,
-
-        ':hover': {
-          color: this.props.brandColor
-        }
-      }, this.props.activeTabStyles)
-    };
+const TabsFactory = ({ type = 'standard', useTabsInMobile, ...props }) => {
+  if (typeof useTabsInMobile !== 'undefined') {
+    console.warn('The useTabsInMobile prop is deprecated and will be removed in a future release.');
   }
-}
-
-const Tab = Radium(TabWithoutRadium);
-
-const StandardTabs = (props) => (
-  <div>
-    {props.tabs.map((tab, index) =>
-      <Tab
-        isActive={props.selectedTab === index}
-        key={tab}
-        onClick={props.onTabSelect.bind(this, index)}
-        styles={{ activeTab: props.activeTabStyles }}
-      >
-        {tab}
-      </Tab>
-     )}
-  </div>
-);
 
-StandardTabs.propTypes = TabsPropTypes;
+  const tabsProps = _pick(props, Object.keys(StandardTabs.propTypes));
+  const TabsComponent = TabsTypes[type];
 
-const PillTabs = (props) => {
-  const styles = {
-    component: {
-      margin: StyleConstants.Spacing.SMALL
-    },
-    tab: {
-      backgroundColor: StyleConstants.Colors.PORCELAIN,
-      borderColor: StyleConstants.Colors.FOG,
-      outline: 'none',
-
-      ':hover': {
-        backgroundColor: props.brandColor,
-        color: StyleConstants.Colors.WHITE,
-        fill: StyleConstants.Colors.WHITE
-      },
-      ':focus': {
-        backgroundColor: props.brandColor,
-        color: StyleConstants.Colors.WHITE,
-        fill: StyleConstants.Colors.WHITE
-      },
-      ':active': {
-        backgroundColor: StyleConstants.adjustColor(props.brandColor, -15)
-      }
-    },
-    selected: {
-      backgroundColor: StyleConstants.Colors.WHITE,
-      color: props.brandColor,
-      cursor: 'default',
-
-      ':hover': {
-        backgroundColor: 'transparent'
-      },
-      ':focus': {
-        backgroundColor: StyleConstants.Colors.WHITE,
-        color: props.brandColor
-      },
-      ':active': {
-        backgroundColor: 'transparent'
-      }
-    }
-  };
+  if (!TabsComponent) throw new Error(`Unknown Tabs type: ${type}`);
```

Comment:
I should have mentioned that in the description. Making `Tabs` more "open" was one of my goals with this refactoring :)